js/app.js

```diff
@@ -363,16 +349,17 @@
 				});
 			}
 		},
+		addParticipantToCall: function(token, participant) {
+			return this.signaling.addParticipantToCall(token, participant);
```

Comment:
just to be sure, this route (addParticipant) is to invite a user to a room, not when a user joins/loads a room. That's why I think it should be in the other one, and the php backend then drops a similar message to the signaling server like on "kick"